source/includes/_lips_events.md

```diff
@@ -146,6 +148,13 @@ curl -L -g -X GET 'https://data-connect-lips.ganettdigital.com/events?global_mas
 |call_recording_url| String | yes | Only included when interaction is call|
 |call_duration| Integer | yes | Length of call in seconds -- only included when interaction is call|
 
+**Booking**
+
+| Field Name | Datatype | Nullable | Description |
+|---|---|---|---|
+|booking_id| String | no | the external id of the booking event |
```

Comment:
the external id (appointment token) of the Appointment. Used Internally to retrieve additional data about the Appointment.